config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  namespace :v1, defaults: { format: :json } do
+    resources :establishment, only: [:index, :create, :show, :update]
```

Comment:
```suggestion
    resources :establishments
```
actually you add all resources on controller, including `destroy` method